content/en/continuous_integration/dora_metrics/_index.md

```diff
@@ -0,0 +1,292 @@
+---
+title: DORA Metrics
+kind: documentation
+description: Learn how to use DORA metrics to improve and measure software development.
+is_beta: true
+further_reading:
+- link: "/continuous_integration/pipelines"
+  tag: "Documentation"
+  text: "Learn about Pipeline Visibility"
+- link: "/tracing/service_catalog"
+  tag: "Documentation"
+  text: "Learn about the Service Catalog"
+- link: "https://github.com/DataDog/datadog-ci"
+  tag: "Source Code"
+  text: "Learn about the `datadog-ci` CLI tool"
```

Comment:
```suggestion
  text: "Learn about the datadog-ci CLI tool"
```
Backticks don't work in Further Reading, they just show up literally. Not sure how you feel about that -- you can leave them in if the visible backticks are better than nothing